tests/zenko_e2e/backbeat/ReplicationUtility.js

```diff
@@ -561,6 +563,57 @@ class ReplicationUtility {
             return cb();
         });
     }
+    
+    assertNoBlob(containerName, key, cb) {
+        this.azure.getBlobProperties(containerName, key, err => {
+            assert(err.code);
```

Comment:
Could be worthwhile to assert the error code `BlobNotFound`.

Reply:
Removed this function as not relevant to AWS